core/src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -466,15 +470,12 @@ private void renderSidebar() {
       }
 
       for (Competitor competitor : sortedCompetitors) {
-        if (!firstTeam) {
+        if (!(firstTeam || (isWool() && competitorsWithGoals.size() > 5))) {
```

Comment:
Make the "isWool()" into a "isHighlyCompact()" or something along the lines, don't use "competitors with goals" as your reason, make it the total rows being > MAX_ROWS even using the normally compact format